src/main/app.ts

```diff
@@ -5,14 +5,20 @@ import cookieParser from 'cookie-parser';
 import express from 'express';
 import favicon from 'serve-favicon';
 
-import { Helmet } from 'main/modules/helmet';
-import { RouterFinder } from 'router/routerFinder';
-import { HTTPError } from 'main/HttpError';
-import { Nunjucks } from 'main/modules/nunjucks';
-import { I18Next } from 'main/modules/i18n';
+import {Helmet} from 'main/modules/helmet';
+import {RouterFinder} from 'router/routerFinder';
+import {HTTPError} from 'main/HttpError';
+import {Nunjucks} from 'main/modules/nunjucks';
+import {I18Next} from 'main/modules/i18n';
+import * as secrets from './modules/secrets';
+import * as appinsight from './modules/appinisght';
 
-const config = require('config');
-const { Logger } = require('@hmcts/nodejs-logging');
+import config from 'config';
+
+secrets.setup(config);
+appinsight.setup(config);
+
+const {Logger} = require('@hmcts/nodejs-logging');
```

Comment:
i think this was from express-template and probably not part of your PR but while you are here, could you update this too please - `import { Logger } from '@hmcts/nodejs-logging';`